man/sd_bus_message_open_container.xml

```diff
@@ -100,8 +100,13 @@
   <refsect1>
     <title>Return Value</title>
 
-    <para>On success, these functions return a non-negative integer. On failure, they return a negative
-    errno-style error code.</para>
+    <para>On success, these functions return a non-negative integer.
+    <function>sd_bus_message_open_container()</function> and <function>sd_bus_message_close_container()</functiony>
+    return 0.
+    <function>sd_bus_message_enter_container()</function> returns 0 if the end of the current container
+    has been reached or 1 on opening a new container.
```

Comment:
This is not wrong, but it's not very clear either. Maybe
> `<function>sd_bus_message_enter_container()</function> returns 1 if it successfully opened a new container, and 0 if that was not possible because the end of the currently open container or message was reached.`
?